libraries/apollo-runtime/src/commonMain/kotlin/com/apollographql/apollo3/ApolloCall.kt

```diff
@@ -15,121 +15,96 @@ import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.toList
 
 /**
- * An [ApolloCall] is a thin class that binds an [ApolloRequest] with its [ApolloClient].
+ * An [ApolloCall] is a thin class that binds an [ApolloRequest] with its [ApolloClient] and offers a fluent way to build an [ApolloRequest].
```

Comment:
Feels a bit off but not sure. Maybe:
```suggestion
 * An [ApolloCall] is a thin class that binds an [ApolloRequest] with its [ApolloClient]. It offers a fluent way to configure the [ApolloRequest].
```